package.json

```diff
@@ -34,7 +35,8 @@
     "react-native-safe-area-context": "3.1.9",
     "react-native-screens": "~2.15.2",
     "react-native-svg": "^12.1.0",
-    "react-native-web": "~0.13.12"
+    "react-native-web": "~0.13.12",
+    "url-otpauth": "^2.0.0"
```

Comment:
is this what we're using in Web? I remember we are using a library but not which one

Reply:
Yeah, I took this lib and the `parse` utility from here. https://github.com/nearform/optic/blob/master/src/lib/qr-parser.js#L19